src/measurements/measurements.ts

```diff
@@ -1,8 +1,8 @@
-import { area as turfArea, booleanEqual, distance as turfDistance, point } from '@turf/turf';
+import { booleanEqual, area as turfArea, distance as turfDistance, type Units } from '@turf/turf';
 import { Geodesic } from 'geographiclib-geodesic';
```

Comment:
Looks like this library isn't widely used and isn't updated, are we sure we want to use it?
Is it used elsewhere in our code? (I can't currently look if it is).

Reply:
if the calculations made by this lib are correct then we might expose wrappers to it's functions,
as they are tested by the lib itself and have no external dependencies.

i'm not in favor of wrapping external libraries functions so we can consider removing it
in any case it's not used anywhere else in our code

Reply:
So if it's not used anywhere else maybe we should remove this dependency.

Reply:
will investigate on distances implementation needed for this lib